KTL.js

```diff
@@ -11487,6 +11493,7 @@ function Ktl($, appInfo) {
                 }
 
                 function hideColumn(viewId, columnIndex) {
+                    $(`#${viewId} tr > th:nth-child(${columnIndex}) i.ktlTooltipIcon`).hide(); // Hide tooltip icons
```

Comment:
Same with this trailing comment. Write a comment to add information that is not in the code. A "why" for example.

Reply:
Okay noted, thanks for advice